data/releases/schedule.yaml

```diff
@@ -2,6 +2,12 @@
 # This file helps to populate the /releases page, and is also parsed to find out the
 # latest patch version for a minor release.
 schedules:
+- release: 1.28
+  releaseDate: 2023-08-15
+  next: 1.28.1
+  cherryPickDeadline: 2023-09-08
+  targetDate: 2023-09-13
+  endOfLifeDate: 2024-10-28
```

Comment:
```suggestion
  endOfLifeDate: 2024-10-28
  previousPatches:
    - release: 1.28.0
      targetDate: 2023-08-15
```

Reply:
Ah yes, thanks, maybe this also needs a change in the docs lead handbook, should I go ahead with doing that too?